### ISSUE TITLE
Extract the Distogram and Other Detailed Outputs using ColabFold

### ISSUE THREAD

**Issue by kazemf78** [sokrypton/ColabFold#305]
Hi,
I wanted to use ColabFold to get the single and pair representation of several proteins along with the distogram. The former can be saved with including `save_single_representation=True` and `save_pair_representation=True` in the parameters of the `run` function in the `AlphaFold2_batch.ipynb` notebook (similar to what you mentioned previously in this [issue](https://github.com/sokrypton/ColabFold/issues/226#issuecomment-1123506156), and by the way, I didn't understand what file or notebook exactly you meant when mentioning `colabfold_batch` executable or script in that issue!). My question now is that how can I also get the distogram output?
For example, the `prediction_result` variable contains several other detailed outputs along with the distogram but seemingly it's not currently exposed to use in `AlphaFold2_batch.ipynb` notebook results (or in any outputs of the functions implemented in `colabfold/batch.py` file).
Best,

**Comment by Abhishaike** [comment]
Bumping this, how would I guess access to the distogram?

**Comment by YoshitakaMo** [comment]
`colabfold_batch` also has the arguments `--save-single-representations` and `--save-pair-representations`, but are they what you want?

**Comment by Abhishaike** [comment]
Those are, as I understand it, just embeddings! I believe me and OP are looking for the more rich set of information that’s present in prediction_result from the Alphafold model run, which includes things like logits and the distogram

**Comment by sokrypton** [comment]
We've added flag --save_all that should extract this info
